src/test/java/Pages/GooglePage.java

```diff
@@ -0,0 +1,71 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class GooglePage {
+
+    private static final By REJECT_ALL = By.id("W0wltc");
+
+
+    private static final By SEARCH_FIELD = By.id("APjFqb");
+    private static final By SEARCH_RESULTS = By.xpath("//a/h3");
+    private static final By FEELING_LUCKY = By.id("gbqfbb");
+    //redirected to url that contains search term
+    private WebDriver driver;
+
+    public GooglePage() {
+        this.driver = new ChromeDriver();
```

Comment:
As mentioned, new ChromeDriver in numerous places, try to remove this and reuse the existing driver, as discussed in Slack